cluster/config-defaults.yaml

```diff
@@ -93,6 +93,7 @@ skipper_validate_query: "true"
 skipper_validate_query_log: "false"
 
 skipper_default_filters: 'disableAccessLog(2,3,404,429) -> fifo(2000,20,"1s")'
+skipper_default_filters_append: 'stateBagToTag("uid", "client.uid")'
```

Comment:
I believe the state bag key is `auth-user` as described in https://github.com/zalando/skipper/blob/master/docs/reference/filters.md#statebagtotag

Also, do we have other meanings for "uid" - maybe we should just use it as a tag name: `stateBagToTag("auth-user", "uid") `

Reply:
I used https://github.com/zalando/skipper/blob/master/filters/auth/tokeninfo.go#L369 , which is of course only the parsed in-memory object, not stateBag. 
Thanks for pointing it out. I see in auth/auth.go on authorized() we actually set  "auth-user" stateBag as documented.

Reply:
> Also, do we have other meanings for "uid" - maybe we should just use it as a tag name: `stateBagToTag("auth-user", "uid") `

I don't know what users would think of "uid" tag. To be more specific I used the client.uid, because it's whatever the client sent to us.